azure/azuredeploy.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "environmentName": {
+      "type": "string"
+    },
+    "keyVaultName": {
+      "type": "string"
+    },
+    "storageAccountName": {
+      "type": "string"
+    },
+    "keyVaultReaderServicePrincipalId": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "location": "[resourceGroup().location]",
+    "tags": {
+      "Environment": "[parameters('environmentName')]",
+      "Parent Business": "Teacher Training and Qualifications",
+      "Portfolio": "Early Years and Schools Group",
+      "Product": "Database of Qualified Teachers",
+      "Service": "Teacher Training and Qualifications",
+      "Service Line": "Teaching Workforce",
+      "Service Offering": "Database of Qualified Teachers"
+    },
+    "tenantId": "9c7d9dd3-840c-4b3f-818e-552865082e16",
+    "tfstateContainerName": "dqtapi-tfstate",
+    "deliveryTeamGroupId": "ce5793a4-b822-4018-a65d-6c04e96918b2"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
```

Comment:
Can we enable blob versioning for security?
I think it's via https://docs.microsoft.com/en-us/azure/storage/blobs/versioning-enable?tabs=template